charts/argo-cd/README.md

```diff
@@ -64,6 +64,17 @@ applicationSet:
   replicas: 2
 ```
 
+## Core mode
+
+You can deploy [Argo-CD core](https://argo-cd.readthedocs.io/en/stable/operator-manual/core/) with this chart by disabling the server and dex components:
+
+```yaml
+server:
+    enabled: false
```

Comment:
given one can already set `https://github.com/argoproj/argo-helm/blob/main/charts/argo-cd/values.yaml#L1928` this to "0". What is this PR achieving?

This isn't how i'd have expected core mode to have been enabled.